godtools/App/Services/Renderer/Views/ChooseYourOwnAdventure/ViewFactory/ChooseYourOwnAdventurePageViewFactory.swift

```diff
@@ -24,16 +24,18 @@ class ChooseYourOwnAdventurePageViewFactory: MobileContentPageViewFactoryType {
         
         if let contentPage = renderableModel as? ContentPage {
             
-            let page: Int = renderedPageContext.page
             let contentInsets: UIEdgeInsets
             let itemSpacing: CGFloat
             
-            if page == 0 {
+            let isIntroPage: Bool = contentPage.id == "intro"
+            let isCategoriesPage: Bool = contentPage.id == "categories"
+            
+            if isIntroPage {
```

Comment:
We really shouldn't be doing custom rendering logic based on a page id. What was the intent behind the custom inset logic here?

Reply:
Hey @frett it's for the leading and trailing for the page content.  I remember I was basing it off Figma and the intro page had greater leading and trailing content insets.

![Simulator Screen Shot - iPhone 8 - 2023-03-06 at 15 00 23](https://user-images.githubusercontent.com/59846460/223218429-58779a31-e014-4223-a42b-118d6344182e.png)


![Simulator Screen Shot - iPhone 8 - 2023-03-06 at 15 00 26](https://user-images.githubusercontent.com/59846460/223218442-135ad2ba-93ce-470e-a062-c69eff18e18a.png)

Reply:
we'll have to revisit some of the layout capabilities of the renderer once we convert the mobile version to SwiftUI/Jetpack Compose